### DISCUSSION TITLE
Feedback of #48615

### DISCUSSION THREAD

**Discussion by ghost** [community/community#54138] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hello @GitHub,Thank you for responding to my report #48615
But why is [this](https://github.com/cirosantilli/china-dictatorship) **didn't delete**?And it is kind of content that incites violence, rumors, politics,GitHub Trust & Safety reply is always:
```
Hi haoming9245,
Thanks for taking the time to let us know. Our team is currently investigating the account in question to determine if their content or conduct violates GitHub's [Terms of Service](https://docs.github.com/site-policy/github-terms/github-terms-of-service).
Please let us know if we can help in any other way.
Regards,  
GitHub Trust & Safety
```
After a few weeks, they didn't reply to anything

First,I don't want open source software to be mixed with any politics,Second, I hope GitHub gets better and better, but it's not, **it's a political tool of the West**

The most important,**I certainly do not like people who oppose humanitarian donations (for Ukraine or Iraq) and create garbage under the corresponding warehouses, because they forcibly drag humanitarianism into politics**

At last,I said to those key politicians in the three languages that the key politicians often use:
**English:Stop, spare the people of every country or region, and don't create political garbage
繁體中文:放下鍵盤，放過每個國家或地區的人民，不要製造政治垃圾
简体中文:放下键盘，放过每个国家或地区的人民，不要制造政治垃圾**
I hope GitHub clean these spam and keep GitHub clean

**Comment by ettaboyle** [comment]
@hhm1  Thank you kindly for reaching out, and sharing this update.  As you have an open issue with our trust & safety team I am going to close this discussion. We are unable to respond publicly on this topic. 

If you are ever experiencing behavior against our Terms of Service or need to report GitHub account-related abuse, you can reach out to our private support team. We are unable to assist with or discuss personal account details here in Discussions.

Please visit this link for more details: https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam 

Thank you for sharing this post and helping flag suspicious content in the GitHub Community Discussions. 
We hope that helps!
